### DISCUSSION TITLE
Newly added mods don’t load on next boot up of the game

### DISCUSSION THREAD

**Discussion by mRama3415** [Raytwo/ARCropolis#438] [Issues] [answered]
When a new mod is added to the mods folder on arc 3.8.0 it doesn’t load on the next boot up into the game. The game needs to be restarted and booted up again for the new mods to appear. The “New mods have been detected. Would you like to enable them?” prompt shows up on the initial boot, but even when hitting yes, they don’t appear. On the second boot, the prompt doesn’t appear, but the mods load successfully.

**Comment by Raytwo** [comment, marked as answer]
[Follow your issue here](https://github.com/Raytwo/ARCropolis/issues/454)
